### ISSUE TITLE
[bug]: setup pull preview version failed

### ISSUE THREAD

**Issue by elmagnificogi** [makeplane/plane#5932]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

if self-hosted use preview version , follow the cmd from offical doc
```
export RELEASE=preview
curl -fsSL https://raw.githubusercontent.com/makeplane/plane/$RELEASE/deploy/selfhost/install.sh | sed  's@BRANCH=master@BRANCH='"$RELEASE"'@' > setup.sh
```
you will get the fail

> Plane supports amd64
Syncing environment variables...
Environment variables synced successfully
Updating custom variables...
Custom variables updated successfully
unknown flag: --policy
Failed to pull the images. Exiting...

i check the real cmd will use 
```
docker-compose pull --policy always
```
in fact the preview cant get all right version

### Steps to reproduce

follow the install plane docker doc
https://developers.plane.so/self-hosting/methods/docker-compose

### Environment

Deploy preview

### Browser

None

### Variant

Self-hosted

### Version

preview

**Comment by bjoernkowarsch** [comment]
Hi @akshat5302 , I just tried to install plane community edition and faced the same problem, so googling brought me here. Is there a way around this issue? And what do you mean by saying _"We recommend setting up a stable release. "_? Isn't the script installing a stable release?

Thank you.

**Comment by akshat5302** [comment]
Yes, @bjoernkowarsch, the script will take care of that task for you. However, the error is due to an outdated version of Docker Compose. Please update Docker Compose and try setting it up again.

**Comment by bjoernkowarsch** [comment]
Thank you @akshat5302 , I was on a snap version of docker in ubuntu, uninstalled that and installed the latest official docker-ce version via apt. That worked for me and made the script install everything.

**Comment by akshat5302** [comment]
Closing this issue
